bin/evaluate

```diff
@@ -13,7 +13,7 @@ import argparse
 import warnings
 
 from madmom.utils import search_files, match_file
-from madmom.evaluation import onsets, beats, notes, tempo, alignment
+from madmom.evaluation import onsets, beats, notes, tempo, alignment, chords
```

Comment:
Ok. Since they were not sorted alphabetically until now, I did not care. I'm going to change that.